src/EventListener/DeserializeListener.php

```diff
@@ -52,7 +52,7 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod(Request::METHOD_DELETE)
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
-            || ($request->isMethod(Request::METHOD_PUT) &&  '' === $requestContent)
+            || ($request->isMethod(Request::METHOD_PUT) && '' === $requestContent)
```

Comment:
As this listener will be executed for every request (even if its a page not managed by API Platform), let's use the fastest alternative.